src/packet/signature.js

```diff
@@ -108,6 +108,9 @@ class SignaturePacket {
 
     // hashed subpackets
     i += this.readSubPackets(bytes.subarray(i, bytes.length), true);
+    if (!this.created) {
+      throw new Error('Invalid signature packet: missing signature creation time in hashed area.');
```

Comment:
This error also feels overly specific to me, I think it could just say
```suggestion
      throw new Error('Missing signature creation time subpacket.');
```
And then in many cases `Could not find valid * signature:` will already be prepended to it.

And then if someone complains that they have a signature with a creation time in the unhashed area, we can point them to https://datatracker.ietf.org/doc/html/draft-ietf-openpgp-crypto-refresh#section-5.2.3.4.